### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by yutyra** [SylEleuth/gruvbox-plus-icon-pack#10] [Q&A]
This is really fantastic! I love these icons, is there any way I could access the icon files so that they could be uploaded as file icons for a Windows system? Sorry if this is not asked well. Thank you!

**Comment by SylEleuth** [comment]
I'm not Windows user and don't know much about that system, but just made a quick look at it and it seems that Windows is using ICO files for icons, which I'm not familiar with. Also, and I might be wrong here, but I think You can't change icons of executables, only to make shortcuts with a new icon. And I have no idea about symbolics and folders. You would probably have to change some registry to do that. And to even start You would have to know what icons you need, what sizes, etc. If it's even doable. I personally don't have interest in doing anything in that direction. Especially as I'm not fond of the Windows OS. 

After reading Your question again - You could access icons by [downloading them and unpacking](https://github.com/SylEleuth/gruvbox-plus-icon-pack#installation). They are in svg vector format which uses xml and is easy accessible. You can do whatever You want with them, including modifying for Your system.

**Reply by yutyra** [reply]
This worked really well for me. I love how these look. It's a bit tedious to do it for Windows but very worth it :) Thank you for creating!!!
